src/cv/apply_detections.py

```diff
@@ -0,0 +1,103 @@
+import numpy as np
+import argparse
+import imutils
+import json
+import pickle
+import cv2
+import os
+import settings
+import time
+import yolo_video_detect as obj_detector
+import recognize_faces as face_recognize
+#import cv_producer
+#import insert_frame
+
+def face_tracker_helper(face_tracker, new_detections):
+    """ Returns True if there is at least 1 new detection """
```

Comment:
You should make a note in the docs that it changes face_tracker. I usually would suggest you to return the newly changed face_tracker as a return value instead of modifying the object by reference. Change the method name as well.

Reply:
Added comment